src/survey/factory.ts

```diff
@@ -9,21 +9,21 @@ import { UUIDs } from '../utils'
 export type SurveyFactoryParams = {
   ownerUuid: string
   name: string
-  label?: string | null
+  label?: string
   languages?: LanguageCode[]
   published?: boolean
   draft?: boolean
   collectUri?: string
-  descriptions?: Labels[]
+  descriptions?: Labels
 }
 
 const defaultProps = {
-  label: null,
+  label: null as any,
   languages: [LanguageCode.en],
   published: false,
   draft: true,
-  collectUri: null,
-  descriptions: null,
+  collectUri: null as any,
+  descriptions: {},
```

Comment:
I think label, descriptions and collectUri should be removed from defaultProps

Reply:
ok